### PR TITLE
MGMT-14881: return readable error when disk no found

### PR DESCRIPTION
**Pull request by eifrach**
* Add readable error when disks are not begin detected 

## List all the issues related to this PR

- [ ] New Feature 
- [x] Enhancement 
- [x] Bug fix
- [x] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by eifrach** [comment]
@avishayt  can you please approve the error message is OK or do you want to rephrase it ?

**Review by avishayt — APPROVED** [review]

File: swagger.yaml [hunk 1]
```diff
@@ -6338,6 +6338,7 @@ definitions:
       - 'no-skip-installation-disk'
       - 'no-skip-missing-disk'
       - 'no-ip-collisions-in-network'
+      - 'no-disk-detected'
```
Comment (gamli75) [inline swagger.yaml:6341]: why do we need a new validation? we can use has-min-valid-disks

File: .gitignore [hunk 2]
```diff
@@ -6,7 +6,7 @@ venv/
 kubevirtci/
 **/RCS/**
 .*.swp
-
+aws/
```
Comment (avishayt) [inline .gitignore:9]: ?

File: swagger.yaml (hunk 1 above)
Comment (avishayt) [reply swagger.yaml:6341]: Yes, it already validates at least one eligible disk.  Just need to make sure that the validation appears in the UI when there are no disks at all.

File: .gitignore (hunk 2 above)
Comment (eifrach) [reply .gitignore:9]: one of the skipper make created this folder - I added it to the ignore so it won't get merged by mistake

File: swagger.yaml (hunk 1 above)
Comment (eifrach) [reply swagger.yaml:6341]: The validation is [here](https://github.com/openshift/assisted-service/blob/master/internal/host/validator.go#L1725-L1730) 

The name of the validation is a bit misleading, This is more to validate disk to format  is existing on the host 
it validate if the ID previously  found is still in inventory.  This looks like for cases of reboot / changed HW.

File: openshift/template.yaml [hunk 3]
```diff
@@ -586,4 +586,4 @@ objects:
                   address:
                     socket_address:
                       address: 127.0.0.1
-                      port_value: 8090
+                      port_value: 8090
```
Comment (gamli75) [inline openshift/template.yaml:589]: why do we need this change?

Comment (eifrach) [reply openshift/template.yaml:589]: look's the same, this may  accrued during the rebase - I'll check it out

**Comment by eifrach** [comment]
closing and reopening  this due to rebase issues 
/close

**Pull request closed without merging**